agent/fw_wordpress.c

```diff
@@ -376,8 +376,9 @@ NR_PHP_WRAPPER(nr_wordpress_wrap_hook) {
 NR_PHP_WRAPPER_END
 
 /*
- * A call_user_func_array() callback to ensure that we wrap each hook function.
+ * PHP 7.3 and below uses old-style wraprec, and will use old style cufa calls.
```

Comment:
Could you elaborate why `add_filter` cannot be wrapped for PHPs < 7.4?

Reply:
This will not be done to limit number of conflicts when syncing oapi branch.

Reply:
For posterity: add_filter can be wrapped for PHPs < 7.4 but is not required as it wraprec lookup for PHPs < 7.4 is not causing increased agent's overhead.